pkgs/top-level/all-packages.nix

```diff
@@ -18825,11 +18825,7 @@ in
 
   mailman-web = with python3.pkgs; toPythonApplication mailman-web;
 
-  mastodon = callPackage ../servers/mastodon {
-    # With nodejs v14 the streaming endpoint breaks. Need migrate to uWebSockets.js or similar.
-    # https://github.com/tootsuite/mastodon/issues/15184
-    nodejs-slim = nodejs-slim-12_x;
```

Comment:
Isn't that a better variant?
```
  mastodon = callPackage ../servers/mastodon {
    nodejs-slim = nodejs-slim-14_x;
```

Reply:
Yeah I thought about that too. I'd still rather leave the node version unspecified as

- Mastodon only specifies a minimum version in their [readme](https://github.com/tootsuite/mastodon/blob/main/README.md), not a maximum version.
- That'd be one more line of code that would need to be maintained.

The old override was due to specific problems with NodeJS 14. I'd rather wait until the next specific problem with a Node version arises before pinning it again.